crates/web-sys/webidls/enabled/RTCPeerConnection.webidl

```diff
@@ -107,7 +107,8 @@ interface RTCPeerConnection : EventTarget  {
   [ChromeOnly]
   attribute DOMString id;
 
-  RTCConfiguration      getConfiguration ();
+  undefined setConfiguration(optional RTCConfiguration configuration);
```

Comment:
```suggestion
  undefined setConfiguration(optional RTCConfiguration configuration = {});
```
Following the spec es exactly as possible, to make it easier to compare it in diffs later.